features/other_b/support.feature

```diff
@@ -21,39 +21,30 @@ Feature: Filing a support request
   When I follow "Support and Feedback"
     And I fill in "Brief summary" with "you suck"
     And I fill in "Your comment" with "blah blah blah"
-    And I fill in "Your email (optional)" with ""
+    And I fill in "Your email (required)" with ""
     And I select "Deutsch" from "feedback_language"
     And all emails have been delivered
     And I press "Send"
+  Then I should see "Email does not seem to be a valid address."
```

Comment:
It looks like the original intent of this test was to fill out the form with all required information and make sure it works. I'd suggest just filling in a valid email address in this scenario right off the bat, since the whole "leave the email field blank and you'll get an error message" situation is tested in `Scenario: Not logged in, with and without email`.